_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_vaapi.cpp

```diff
@@ -98,7 +99,23 @@ namespace UMC_HEVC_DECODER
             }
         }
     }
+    void PackerVAAPI::PackProcessingInfo(H265DecoderFrameInfo * sliceInfo)
+    {
+        UMC::VideoProcessingVA *vpVA = m_va->GetVideoProcessingVA();
+        if (!vpVA)
+            throw h265_exception(UMC::UMC_ERR_FAILED);
+
+        UMC::UMCVACompBuffer *pipelineVABuf;
```

Comment:
From surrounding lines I anticipate, that following code would made intent clearer.
```suggestion
        UMC::UMCVACompBuffer *pipelineVABuf;
        GetParamsBuffer(m_va, &pipelineBuf);
```